docs/examples/pcrglobwb.ipynb

```diff
@@ -2,57 +2,130 @@
  "cells": [
```

Comment:
<p>directory should be <code>"./parameter-sets/pcrglobwb_rhinemeuse_30min/forcing"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>